nilearn/tests/test_signal.py

```diff
@@ -133,6 +143,8 @@ def test_butterworth():
                                 low_pass=low_pass, high_pass=high_pass,
                                 copy=True)
     np.testing.assert_almost_equal(data, data_original)
+    np.testing.assert_(id(out1)!=id(data_original))
```

Comment:
Python coding style (known as "PEP8") says that you should have a space before and after the "!=" operator.